backend/backend/celery.py

```diff
@@ -3,6 +3,9 @@
 from celery import Celery
 from celery import current_app
 from celery.signals import after_task_publish
+from django.conf import settings
+
+LEDGER = getattr(settings, 'LEDGER', None)
```

Comment:
Be careful here as l.11 set the settings to `backend.settings.prod`. We may need to load `LEDGER` after this.

Reply:
Would you suggest moving line 6 and 8 after line 11?

Reply:
Yes :)

Reply:
done https://github.com/SubstraFoundation/substra-backend/pull/300/commits/53502bd551d46f60d618355bdc165d5b727b604a